src/qutip_qip/device/cavityqed.py

```diff
@@ -255,3 +128,152 @@ def load_circuit(self, qc, schedule_mode="ASAP", compiler=None):
         )
         self.global_phase = compiler.global_phase
         return tlist, coeff
+
+
+class CavityQEDModel(Model):
+    """
+    The physical model for a dispersive cavity-QED system.
```

Comment:
```suggestion
    The physical model for a dispersive cavity-QED system.
```
shall we change "system" with "processor"? Is there any other detail to add?

Reply:
Let's add brief details about the model Hamiltonian (should be the Tavis-Cummings Hamiltonian).